### DISCUSSION TITLE
ERROR FROM UPSCAYL

### DISCUSSION THREAD

**Discussion by Tracee1234** [community/community#148234] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Apologies in advance as I am a senior user of technology and not all that great with a lot of it.
I was using Upscayl and it had an error and somehow I ended up here. Is that correct? Are GitHub and Upscayl directly linked?
I cannot get Upscayl to work on my PC. I have uninstalled, reinstalled, upgraded and I cannot seem to get past the error below:

"VKCreateInstance failed -9"

I am so confused with all the techy stuff that once I seem to get my head around it, the knowledge has been superseded.
Thank you if you can help. Sorry if I have ended up in the wrong place.
Trace :)

![Screenshot_158](https://github.com/user-attachments/assets/f171daae-de0b-4adb-a876-0e631337dfa6)

**Comment by divyamlathiya** [comment]
No, GitHub and Upscayl are not directly linked. GitHub is a platform for hosting code, while `Upscayl` is an AI-based upscaling tool. The issue you're facing with `"VKCreateInstance failed -9"` is related to Vulkan (a graphics API) on your PC, which is likely due to an issue with your graphics drivers or system configuration.

## Here's what you can try to fix it:

**1. Update your graphics drivers** (for NVIDIA, AMD, or Intel).
**2. Ensure Vulkan support is enabled on your system. You can download the Vulkan SDK from the official site if needed.**
**3. Check your GPU compatibility with Vulkan.**
**4. Reinstall Vulkan runtime libraries (if not installed).**
If this doesn’t help, check if your PC meets the system requirements for Upscayl or try running it on a different machine.

Hope this helps!

**Comment by Tracee1234** [comment]
This is so much more information than I could have asked for.

Thank you so much. I will try these steps.

I appreciate your prompt response at this time of year more than you know.

Happy New Year!
Trace :)



Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
